include/geolib/HeightMapNode.h

```diff
@@ -1,5 +1,5 @@
-#ifndef FAST_SIMULATOR_HEIGHT_MAP_H_
-#define FAST_SIMULATOR_HEIGHT_MAP_H_
+#ifndef GEOLIB_HEIGHT_MAP_H_
+#define GEOLIB_HEIGHT_MAP_H_
```

Comment:
Would need `_NODE` in there as well, wouldn't it?

Reply:
@alberth Done